### ISSUE TITLE
Split ztree-dir and ztree-diff to different projects

### ISSUE THREAD

**Issue by fourier** [fourier/ztree#32]
Since the ztree-dir was just a proof-of-concept for ztree-diff, but it keeps its own dedicated users, probably it is better to split these 2 projects and develop ztree-dir as a stand-alone project with more intensive pace to add more features (like nerd-tree for vim or stand-alone x-tree dos app )?

**Comment by rchar01** [comment]
I was looking for a module that, apart from the usual directory tree, would also have the option of showing differences in trees. Therefore, even if the project were divided I would use dir and diff.

From what I noticed ztree (dir and diff) has common parts.
Is it beneficial to separate (duplicate) these parts?

**Comment by fourier** [comment]
It will make it easier to add different features to different projects, without caring how it affect another project. Currently they are quite tightly coupled so its not that easy to do.

**Comment by rchar01** [comment]
So there will be two packages for the elpy and melpy (e.g. ztree-dir and ztree-diff), right?
Will the ztree project on github also be divided, or there will be one repository?

**Comment by fourier** [comment]
No, I think it will still be the same package, just internally they could be splitted.

**Comment by fourier** [comment]
I consider deploying to GNU ELPA as a release and rolling on MELPA as a "beta".
